echidna/test/test_store.py

```diff
@@ -221,10 +216,12 @@ def test_serialisation_fit_results(self):
         self.assertTrue(numpy.array_equal(fit_results.get_raw_stats(),
                                           loaded.get_raw_stats()),
                         msg="Original _stats does not match loaded _stats")
-        self.assertTrue(numpy.array_equal(fit_results.get_penalty_terms(),
-                                          loaded.get_penalty_terms()),
-                        msg="Original _penalty_terms "
-                        "does not match loaded _penlty_terms")
+        for par in fit_results._fit_config.get_pars():
+            self.assertTrue(numpy.array_equal(
+                    fit_results.get_penalty_terms(par),
+                    loaded.get_penalty_terms(par)),
```

Comment:
These are the lines giving the pep8 errors. Could you just check the indentation on 221-224 please, as it doesn't look quite right.